src/ApiTestCase.php

```diff
@@ -87,6 +92,12 @@ public function setUpDatabase()
         }
     }
 
+    public function setUp()
+    {
+        parent::setUp();
+        $this->fixtureLoader = new Loader();
```

Comment:
I would put it together with database setup or create setUpLoader method to keep consistency.